addons/account/security/account_security.xml

```diff
@@ -32,6 +32,11 @@
         <field name="category_id" ref="base.module_category_hidden"/>
     </record>
 
+    <record id="group_discount_per_so_line" model="res.groups">
```

Comment:
IMHO for the sake of consistency the name should be changed also:

group_discount_per_so_line

by

group_discount_per_line

The reason: It impacts sales (if installed) and Account (which sale depend of it).

Regards.